aptos-move/aptos-abstract-gas-usage/src/visitor.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_gas_algebra::{Expression, GasExpressionVisitor};
+
+/// Visitor to traverse the Reverse Polish Notation
+pub struct CalibrationVisitor {
```

Comment:
Please reuse the visitor you implemented in `abstract_algebra.rs`. You can name that one `DynamicExpressionBuilder` or something similar